napps/kytos/of_l2ls/main.py

```diff
@@ -51,6 +52,9 @@ def handle_packet_in(self, event):
 
         switch.update_mac_table(ethernet.source, in_port)
 
+        if packet_in.reason != PacketInReason.OFPR_NO_MATCH:
+            return
```

Comment:
Why not?
I could be wrong, but as I see it, those messages should be treated, but not by of_l2ls... Is it meant to be flooded or the case of a flow_mod? I think not, that's why I made the function ignore it. Am I wrong?
As I see it, those messages should be treated by some other function or some other napp.